docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+      var message = this.props.message;
+
+      render: function () {
+        return (
+          <li key={message.id}>
+            {message.username} - {message.text} <br />
+            <button onClick={this.nearestOwnerProps.deleteMessage}>Delete</button>
+            <button onClick={this.nearestOwnerProps.updateMessage}>{this.state.editing ? 'Cancel' : 'Edit'}</button>
+          </li>
+        );
+      }
+    });
+
+    module.exports = Message;
+```
+
+Some of this should look familiar as the `createOwneeClass` is an implementation on top of the `React.createClass` method. Which means that it has access to all of the same [lifecycle](http://facebook.github.io/react/docs/component-specs.html) methods as a normal `React` class. Additionally, it requires a `render` method just like any other `React` component.
+
+You can also see that we are using `React's` [JSX](http://facebook.github.io/react/docs/displaying-data.html) syntax here. We are able to do this because when we require in our `tux/React` module we assign it to a variable name `React`. JSX automatically translates into `React.METHOD NAME`, thus by writing `var React = require('tux/React')` we can leverage JSX syntax.
+
+If you are unfamiliar with `React` components please review their [docs](http://facebook.github.io/react/). Below is the API documentation for the `createOwneeClass` method.
+
+***
+
+### <a id="React-createOwneeClass"></a>3) React.createOwneeClass [#](#React-createOwneeClass)
+Returns an `OwneeClass` based on the passed in `owneeClassProps`
+
+```javascript
+    var OwneeClass = React.createOwneeClass(owneeClassProps);
+```
+
+#### <a id="createOwneeClass-owneeClassProps"></a>Parameter - `owneeClassProps` - type: OBJECT - required [#](#createOwneeClass-owneeClassProps)
+Properties of the `OwneeClass` to be created. Expected keys:
+
+##### <a id="owneeClassProps-render"></a>Property - `owneeClassProps.render` - type: FUNCTION - required [#](#owneeClassProps-render)
+As with any other `React` class, a [render](#http://facebook.github.io/react/docs/component-specs.html#render) method is required. This method is responsible for returning a `React Element` which can be rendered onto the DOM.
+
+```javascript
+    owneeClassProps.render = function () {
+      return (
+        <div>
+          <h1>hello world</h1>
+        </div>
+      );
+    };
+```
+
+##### <a id="owneeClassProps-LIFECYCLE_AND_SPECS"></a>Methods/Properties - `owneeClassProps.LIFECYCLE_AND_SPECS` - type: FUNCTIONs - optional [#](#owneeClassProps-LIFECYCLE_AND_SPECS)
+As with any other `React` class, the `owneeClassProps` can optionally include `React` [lifecycle methods and specs](#http://facebook.github.io/react/docs/component-specs.html).
+
+```javascript
+    owneeClassProps.componentDidMount = function () {
+      roomActions.get();
+    };
+```
+
+##### <a id="owneeClassProps-nearestOwnerProps"></a>Property - `owneeClassProps.nearestOwnerProps` [#](#owneeClassProps-nearestOwnerProps)
```

Comment:
Hmm I still think this is a kind of confusing place to put this since it looks like it's a property you need to pass in to `owneeClassProps`. Can we move this section under the `Return` section?